Source/EasyNetQ/Persistent/PersistentChannel.cs

```diff
@@ -69,6 +78,8 @@ public PersistentChannel(in PersistentChannelOptions options, IPersistentConnect
 
                 if (exceptionVerdict.Rethrow)
                     throw;
+
+                logger.ErrorException("Failed to invoke channel action", exception);
```

Comment:
I am not sure whether is should be error level, but for the consumer side error level is used.

Reply:
It looks like warning level is better option here, because we do retry this exceptions.